example-react-app/package.json

```diff
@@ -4,10 +4,10 @@
   "dependencies": {
```

Comment:
Please change commit message to `fix: {bug description} {ticket number}` where `{bug description}` is description of the effect (not the cause).